_pages/documentation/learning_gem5/part1/building.md

```diff
@@ -23,7 +23,7 @@ On Ubuntu, you can install all of the required dependencies with the
 following command. The requirements are detailed below.
 
 ```
-sudo apt install build-essential git m4 scons zlib1g zlib1g-dev libprotobuf-dev protobuf-compiler libprotoc-dev libgoogle-perftools-dev python-dev python
+sudo apt install build-essential git m4 scons zlib1g zlib1g-dev libprotobuf-dev protobuf-compiler libprotoc-dev libgoogle-perftools-dev python-dev libboost-all-dev python
```

Comment:
Boost shouldn't be a requirement. We need to fix this, if it is.

Reply:
Ok. I've seen the Jira. I'll get on this eventually. I'll keep this change in the pages for the meantime until this is resolved.

Reply:
Sounds good.